src/components/vm/consoles/consoles.jsx

```diff
@@ -213,15 +213,18 @@ export const ConsoleCard = ({ state, vm, config, onAddErrorNotification, isExpan
         );
     }
 
+    const urlOptions = { name: vm.name, connection: vm.connectionName };
+    const path = isExpanded ? ["vm"] : ["vm", "console"];
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2229-eaaa9181-20250728-131638-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)